droidlet/tools/hitl/dashboard_app/backend/dashboard_server.py

```diff
@@ -85,5 +88,30 @@ def get_interaction_sessions(batch_id):
     emit(DASHBOARD_EVENT.GET_INTERACTION_SESSIONS.value, sessions)
 
 
+@socketio.on(DASHBOARD_EVENT.GET_INTERACTION_SESSION_LOG.value)
+def get_interaction_session_log(id_info_json):
+    """
+    get interaction job session log specified by the id info
+    - input: infomation about id in a json format:
+        {
+            "batch_id": <batch id>,
+            "session_id": <session id>
+        }
+    - output: if the session log can be found, return the content of session log, otherwise return an error code
+    """
+    print(f"Request received: {DASHBOARD_EVENT.GET_INTERACTION_SESSION_LOG.value}")
```

Comment:
nit: we should consider use logging as we are getting more serious.

Reply:
Make sense, I will create a new pr to address this issue